### PR TITLE
implement webpack versioning

### PR DESCRIPTION
**Pull request by Davidmattei**

### CODE REVIEW

File: Resources/doc/example.md [hunk 1]
```diff
@@ -9,9 +9,11 @@ Your html page needs the following **3 elements** for getting a EMSForm.
 ### form.js
 Includes the javascript for sending and receiving postMessage from or to the ems form skeleton.
 ```html
-<script src="https://emsforms-skeleton.test/bundles/emsform/js/form.js"></script>
+<script src="https://emsforms-skeleton.test/bundles/emsform/js/form.{hash}.js"></script>
 ```
 
+The hash can be found in the manifest.json: https://emsforms-skeleton.test/bundles/emsform/manifest.js
```
Comment (dameert) [inline Resources/doc/example.md]: Isnt the Reason of existance of the manifest to prevent using hashes in code?

Comment (dameert) [reply Resources/doc/example.md]: The manifest is the implementation of versioning

**Review by dameert — DISMISSED** [review]
The manifest is good. But you should not use hash from Your html of twig files. There is a way to tell the system to go via the manifest

File: Resources/doc/example.md (hunk 1 above)
Comment (Davidmattei) [reply Resources/doc/example.md]: https://github.com/ems-project/EMSCommonBundle/pull/41

**Pull request merged**